### PR TITLE
SCHOOL-132/Add methods, steps for creating admins, positive and negative test cases, and logs in many classes

### PR DESCRIPTION
**Pull request by davbalabekyan**
@mkrtichstepanyan @HeghineKhachatryan

### CODE REVIEW

File: src/test/resources/features/super_admin/functionality/create_admin.feature [hunk 1]
```diff
@@ -0,0 +1,90 @@
+Feature: Super admin page/ Admin section functionality
+  This feature will cover admin section of super admin page, especially: creating new,
```
Comment (HeghineKhachatryan) [inline src/test/resources/features/super_admin/functionality/create_admin.feature]: add "admins" word in sentence "creating new"

File: src/main/java/com/epam/pages/popup/CreatePopup.java [hunk 2]
```diff
@@ -113,49 +120,63 @@ public boolean checkAllInputFieldsAreEmpty() {
...
     public boolean checkGeneratePasswordButtonIsEnabled() {
+        logger.info("Check 'Generate password' button is active");
         return generatePasswordButton.isEnabled();
     }
 
     public boolean checkGeneratedPasswordStructure() {
+        logger.info("Check generated password structure");
         return passwordInput.getDomProperty("value")
-                .matches("(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$#!%*?&])[A-Za-z\\d@$!#%*?&]{9,50}");
+                .matches("(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$#!%*?&])[A-Za-z\\d@#$%&*()]{9,50}");
```
Comment (HeghineKhachatryan) [inline src/main/java/com/epam/pages/popup/CreatePopup.java:139]: In your regex you can also add other symbols, e.g. `~!^-_?+=|

Comment (davbalabekyan) [reply src/main/java/com/epam/pages/popup/CreatePopup.java:139]: This regex has been written based on the requirement in the confluence page

File: src/test/resources/features/super_admin/functionality/create_admin.feature (hunk 1 above)
Comment (davbalabekyan) [reply src/test/resources/features/super_admin/functionality/create_admin.feature]: Ok, Now I will change it.

**Review by mkrtichstepanyan — APPROVED** [review]

**Pull request merged**